digitalocean/baseapi.py

```diff
@@ -171,7 +171,14 @@ def get_data(self, url, type=GET, params=None):
             msg = [data[m] for m in ("id", "message") if m in data][1]
             raise DataReadError(msg)
 
-        # If there are more elements available (total) than the elements per 
+        # Add the account requests/hour limit
+        self.ratelimit_limit = req.headers.get('Ratelimit-Limit', None)
+        # Add the account requests remaining
+        self.ratelimit_remaining = req.headers.get('Ratelimit-Remaining', None)
+        # Add the account requests limit reset time
+        self.ratelimit_reset = req.headers.get('Ratelimit-Reset', None)
```

Comment:
Move these lines (174->179) to a specific function that will just take care of that

Reply:
Done :)